bench/utils/__init__.py

```diff
@@ -0,0 +1,85 @@
+import struct
+import sys
+import time
+import random
+
+_RANDOM = random.Random(0)
```

Comment:
Yes, I used seed 0 for testing purposes, we can go with whatever seeding method you think is best. The biggest reason I saw for using a seeded RNG is for a better comparison between two benchmarks run on one machine rather than inter-machine similarities so using a randomly generated key will be fine.